drivers/storage/portworx/util/util.go

```diff
@@ -450,6 +464,11 @@ func GetPortworxVersion(cluster *corev1.StorageCluster) *version.Version {
 				pxVersion, err = version.NewSemver(pxVersionStr)
 				if err != nil {
 					logrus.Warnf("Invalid PX version %s extracted from annotation: %v", pxVersionStr, err)
+					pxVersionStr = getPortworxVersionFromManifestURL(manifestURL)
```

Comment:
Just curious, why/where exactly do we need to get the px version from the manifest?

Reply:
because when we doing the upgrade we just set the env var `PX_RELEASE_MANIFEST_URL` to `"https://edge-install.portworx.com/2.12/version"`, the oci monitor image will be sha_sha, currently operator cannot get the version correctly without this.

Reply:
If the version is sha_sha, normally this function returns a very high valid sem version. So any check against 2.12 should have passed and we should have enabled the ccm-go.  So, ideally we should not have needed this. Anyway, if it unblocks you for now, it is okay.

Reply:
seems yes, alright i got it, previously I didn't set it to run ccm-go if the version is invalid, for now without this it should work as well, but adding this shouldn't hurt.